packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.constants.js

```diff
@@ -5,42 +5,50 @@ export const CATEGORIES = [
     category: 'balanced',
     title: 'General Purpose',
     pattern: /eg/,
+    isAvailableFeatureCategory: true,
```

Comment:
why its not added to feature availability API? same question for other flavours.

Reply:
Same response as this :
https://github.com/ovh/manager/pull/6083#discussion_r775806945

Reply:
Completed.